### PR TITLE
Add third party tools

### PR DESCRIPTION
**Pull request by dessalines**
Pic:

Screenshot_20240227_092611_Via_1

### CODE REVIEW

**Comment by Nutomic** [comment]
- I would specifically call it "Moderation tools".
- LemmyAutomod seems to have to most features so I would put it at the top.
- lemmy-bot is a library so it fits better on the left with other libraries
- For Fediseer I dont see any setup instructions

**Comment by wiki-me** [comment]
There are a ton of lemmy tools and software , adding all of them could clutter the page so i think it will be best to keep a select list of projects on the page and to add something like "for a community maintained list of lemmy related software see [awesome-lemmy](https://github.com/dbeley/awesome-lemmy)"

**Comment by dessalines** [comment]
API library means language specific lemmy API, not just any library.

I *could* add a separate block for Moderation-specific tools, but I don't see why we shouldn't list all 3rd-party tools.

I'd also have no problem adding all those tools if someone wants to, we don't have to worry about visual clutter.

**Comment by wiki-me** [comment]
It's not just the visual clutter, the more projects will be added the less chance there is people will check them out (it's [choice overload](https://en.wikipedia.org/wiki/Overchoice).

Also there used to be a list like this on the readme , it changed to a link to awesome-lemmy due to not adding more maintainance burden as new projects keep getting added iirc.

It's also a lot easier to edit then the website, which is why it does not have has many projects probably.

**Comment by Nutomic** [comment]
My idea was to give an overview of external moderation tools which exist for Lemmy, to mitigate the lack of builtin moderation tools. But the list as implemented in this PR doesnt fulfill that purpose. I think it makes more sense as a blog post, then we also dont have to worry about keeping it up to date.

**Comment by dessalines** [comment]
Would renaming that block to moderation tools, and only linking those be good enough?

**Comment by Nutomic** [comment]
Yes that would make more sense. For Fediseer you should link [this](https://gui.fediseer.com/).

File: src/shared/components/app-definitions.ts [hunk 1]
```diff
@@ -67,6 +67,37 @@ export const API_LIBRARIES: ApiLibrary[] = [
   },
 ];
 
+export const MODERATION_TOOLS: ToolDetails[] = [
+  {
+    name: "lemmy-bot",
+    link: "https://github.com/SleeplessOne1917/lemmy-bot",
+    description: "A bot library for Lemmy, the fediverse link aggregator.",
+  },
```
Comment (Nutomic) [inline src/shared/components/app-definitions.ts:75]: Remove this as its a library and not a mod tool.

File: src/shared/components/app-definitions.ts [hunk 2]
```diff
@@ -67,6 +67,37 @@ export const API_LIBRARIES: ApiLibrary[] = [
...
+  {
+    name: "lemmy-bouncer",
+    link: "https://github.com/SleeplessOne1917/lemmy-bouncer",
+    description:
+      "Lemmy automod bot to prevent unvetted users from posting or commenting in a community.",
+  },
```
Comment (Nutomic) [inline src/shared/components/app-definitions.ts:81]: I would move this to the bottom as it as very limited functionality and use case compared to others.

**Review by Nutomic — APPROVED** [review]

**Pull request merged**